src/js/components/RangeInput/StyledRangeInput.js

```diff
@@ -138,6 +139,26 @@ const trackColorStyle = (props) => {
   `;
 };
 
+const disabledRangeInputThumbStyle = (props) => css`
```

Comment:
One thought is to combine these two functions and add one more argument to the function:

```
const disabledRangeInputStyle = (props, context) => css`
  ${disabledStyle(props.theme.rangeInput.disabled.opacity)}
  ${props.theme.rangeInput.disabled.[context] &&
  props.theme.rangeInput.disabled[context].color &&
  `background: ${normalizeColor(
    props.theme.rangeInput.disabled[context].color,
    props.theme,
  )};`}
`;
```